boards/FSM/Src/App/App_FlowMeter.c

```diff
@@ -1,17 +1,28 @@
-#include <stddef.h>
-#include <assert.h>
+#include "App_CanTx.h"
 #include "App_FlowMeter.h"
 
+extern struct World *world;
```

Comment:
We should never really use `world` as a global variable like this, i'll probably looking into a static variable in `main.c`. Also, this doesn't even seem to be used in this .c file anyway.

Reply:
My bad this was left from previous code. I got rid of it 🅱️